packages/mds-db/policies.ts

```diff
@@ -106,9 +133,24 @@ export async function readPolicy(policy_id: UUID): Promise<Policy> {
   throw new NotFoundError(`policy_id ${policy_id} not found`)
 }
 
+async function throwIfRulesAlreadyExist(policy: Policy) {
+  await Promise.all(
+    policy.rules.map(async rule => {
+      const other_policies = await readPolicies({ rule_id: rule.rule_id })
+      other_policies.map(other_policy => {
+        if (other_policy.policy_id !== policy.policy_id) {
+          throw new ConflictError(`Policy containing with a rule of id ${rule.rule_id} already exists`)
+        }
```

Comment:
We typically use `camelCase` as opposed to `snake_case` for variables, unless they're based off of DB columns or query parameters.